### ISSUE TITLE
Question about the validation step during training

### ISSUE THREAD

**Issue by 1713mz** [cqylunlun/GLASS#40]
Hello, I checked the trainer function of the glass class used for the training, and I am a bit confused about how you validate the model during training. Do you validate the model **directly on the real test data** of each class (e.g. from MVTec) and then save the ckpt with the best performance?

**Comment by cqylunlun** [comment]
Yes, this is a common paradigm in anomaly detection tasks. We borrowed the code for this part from [SimpleNet](https://github.com/DonaldRR/SimpleNet/blob/351a2b8d4e8cfc944dbccbf9bc6ceda930c6f26b/simplenet.py#L419-L444). Since there are relatively few anomalous samples in the dataset and no separate validation set is provided, the test set is directly used as the validation set.
